go.sum

```diff
@@ -765,10 +765,15 @@ go.mongodb.org/atlas v0.7.2 h1:wB3+hP71t3mK+JOSrjBFbrzb5MsZRzDtZlpEKp58KK0=
 go.mongodb.org/atlas v0.7.2/go.mod h1:CIaBeO8GLHhtYLw7xSSXsw7N90Z4MFY87Oy9qcPyuEs=
 go.mongodb.org/atlas v0.7.3-0.20210302081736-7ccf7e1f4351 h1:3NuN2K6VtDuDNfAKX/4j6H4pLaT8LIkvfXD3F7Oy30c=
 go.mongodb.org/atlas v0.7.3-0.20210302081736-7ccf7e1f4351/go.mod h1:CIaBeO8GLHhtYLw7xSSXsw7N90Z4MFY87Oy9qcPyuEs=
+go.mongodb.org/atlas v0.8.0/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
 go.mongodb.org/atlas v0.8.1-0.20210528214129-330aa9199dbf h1:B9IN5pGOtgtgmx8jjtuP9Gx6PPFWIIKx3AKQKYKcxJM=
 go.mongodb.org/atlas v0.8.1-0.20210528214129-330aa9199dbf/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
 go.mongodb.org/atlas v0.8.1-0.20210604212331-3f489d7a4eed h1:qPaT/RvfXXKEJ2fTNckgPKoqfg4oLgJFj7exXUM8mUQ=
 go.mongodb.org/atlas v0.8.1-0.20210604212331-3f489d7a4eed/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
+go.mongodb.org/realm v0.0.0-20210604190232-4d3dfc67ed71 h1:2jY3qOwzS8AQMRtuGxs8y/k1FRGWQHfvrrtEk0X+61Y=
+go.mongodb.org/realm v0.0.0-20210604190232-4d3dfc67ed71/go.mod h1:Tru1+aHka6+4uXYigPKoNySbNWgtkfsanVddwmcqvOM=
+go.mongodb.org/realm v0.0.0-20210618220639-e70c919266f2 h1:3ukcBKIOun3QRMZzdVRCU/7uVhjR7XQsdD/cCH5V8So=
+go.mongodb.org/realm v0.0.0-20210618220639-e70c919266f2/go.mod h1:Tru1+aHka6+4uXYigPKoNySbNWgtkfsanVddwmcqvOM=
```

Comment:
[q] Why does go.sum mentions 2 versions of `go.mongodb.org/realm`? When only `go.mongodb.org/realm v0.0.0-20210618220639-e70c919266f2` is present in go.mod?

Reply:
That looks like `go mod tidy` might need to be run to remove the duplicates, unless the reference is coming in from another lib.

Reply:
good eye @threebee , just pushed with vendor updated, thank you